spec.bs

```diff
@@ -115,6 +115,7 @@ dictionary DocumentPictureInPictureOptions {
   [EnforceRange] unsigned long long width = 0;
   [EnforceRange] unsigned long long height = 0;
   boolean disallowReturnToOpener = false;
+  boolean disallowPositionReuse = false;
```

Comment:
How about `resetPosition` or `ignorePreviousPosition`?

Reply:
`ignorePreviousPosition` -- i agree that this is clearer that the current one, though see below.

`resetPosition` is a little confusing to me -- it sounds to me like we're asking the UA to reset some well-defined state rather than give it a hint.

Orthogonally to that, while writing this up, I started disliking using the word "position".  The docs say "position and size" everywhere, but the parameter is just "position".  "disallowBoundsReuse" was what I was considering, but it seemed fairly opaque what "bounds" it might be talking about.  I didn't use it as a result.

The combination of these, `ignorePreviousWindowBounds`, however, I like quite a lot.  I'll try this one.